### PR TITLE
Add service message to service manager home page

### PR DESCRIPTION
**Pull request by jon-shipley**
- [x] add serviceMessage to service-manager home page
- [x] allow paragraphs in service message
- [x] add message preview when setting the message
- [x] black text in the service message

### CODE REVIEW

File: admin/package.json [hunk 1]
```diff
@@ -20,17 +20,17 @@
     "start": "yarn build && node --max-http-header-size 15000 ./dist/server.js",
     "start:dev": "yarn dev-build && concurrently -r 'nodemon --inspect --delay 1500ms --watch ./dist --max-http-header-size 15000 ./dist/server.js' 'gulp watch'",
     "test": "yarn lint && yarn jest",
-    "test:watch": "yarn jest --watch --coverage=no",
-    "test:nocoverage": "yarn lint && yarn jest --coverage=no",
-    "test:coverage": "yarn lint && yarn jest --coverage=yes",
+    "test:watch": "yarn jest --watch",
+    "test:nocoverage": "yarn lint && yarn jest",
```
Comment (GuyHarwood) [inline admin/package.json:24]: i'm a bit confused, as `test:nocoverage` appears to be the same as just `test`
are you doing this on purpose for compatibility purposes with any existing usages in the CI pipeline?

**Review by GuyHarwood — COMMENTED** [review]
LGTM.  just one question regarding the yarn scripts

File: admin/package.json (hunk 1 above)
Comment (jon-shipley) [reply admin/package.json:24]: Tbh, I don't think anything is actually using `test:nocoverage`.  I think the default, of no coverage, is perfect during development.  Shall I just remove `test:nocoverage`?

Comment (GuyHarwood) [reply admin/package.json:24]: yes please do.

**Review by GuyHarwood — APPROVED** [review]
LGTM
great to see the sanitizer and various tests 💯

**Pull request merged**